assignment2/hex_calculator/TypeConversion.java

```diff
@@ -0,0 +1,14 @@
+public class TypeConversion 
+{
+	int hexToInt(String strHex)
```

Comment:
Since this works are a Utility class. The functions can be made static. Read more about static functions and their usage.